aptihramy-frontend/src/components/DisplayPeople.vue

```diff
@@ -6,71 +6,87 @@
 
         <!-- Table Header -->
         <v-row class="table-header-row">
-            <v-col class="table-header-text" v-for="(column, index) in COLUMNS" :key="index">
+            <v-col class="table-header-text" v-for="(column, index) in COLUMNS_PRETTY" :key="index">
                 {{ column }}
             </v-col>
         </v-row>
+
         <div class="table-body">
-            <template v-for="(record, recordIndex) in filtredData" :key="recordIndex">
-                <v-row :class="['table-data-row', { 'table-alternate-data-row': recordIndex % 2 === 0 }]">
-                    <v-col class="table-data-text" v-for="(column, index) in COLUMNS" :key="index">
-                        {{ record[COLUMN_TRANSLATION.get(column)] }}
+            <template v-for="(record, recordIndex) in filteredData" :key="recordIndex">
+                <v-row :class="['table-data-row', { 'table-alternate-data-row': recordIndex % 2 === 0 }]"
+                    @click="handleRowClick(record.index)">
+                    <v-col class="table-data-text" v-for="(column, index) in COLUMNS_PRETTY" :key="index">
+                        {{ record.value[COLUMN_PRETTY_TO_RAW.get(column) as string] }}
                     </v-col>
                 </v-row>
             </template>
         </div>
-
     </v-card>
 </template>
 
-<script setup>
-import { ref, computed, defineProps, watch } from 'vue';
-import { COLUMN_TRANSLATION, COLUMNS } from '@/config/constants';
-import { TEST_DATA, TEST_DATA_1805, FIRST_RECORDS } from '@/config/test_data';
-import '../styles/table.css'
-const props = defineProps({
-    selectedColumnsRows: {
-        type: Map,
-        default: new Map()
-    }
-});
+<script setup lang="ts">
+import { computed } from 'vue';
+import { COLUMN_PRETTY_TO_RAW, COLUMNS_PRETTY } from '@/config/constants';
+import { FIRST_RECORDS } from '@/config/test_data';
+import { useRouter } from 'vue-router';
+import '../styles/table.css';
+import '../styles/theme.css';
+
+// Define the shape of a record
+interface ValueType {
+    [key: string]: string | number
+}
+// Define the full object that includes the 'value' and 'index' fields
+interface RecordType {
```

Comment:
I think it would be a good idea to regroup all the types/interfaces in a single file (ex: `types.ts`)